src/main/resources/application.properties

```diff
@@ -32,23 +27,16 @@ app.env.unleashUrl=${UNLEASH_API_URL:null}
 app.env.ytelseskontraktV3Endpoint=${VIRKSOMHET_YTELSESKONTRAKT_V3_ENDPOINTURL:null}
 app.env.varselOppgaveV1Endpoint=${VARSELOPPGAVE_V1_ENDPOINTURL:null}
 app.env.behandleArbeidssoekerV1Endpoint=${VIRKSOMHET_BEHANDLEARBEIDSSOEKER_V1_ENDPOINTURL:null}
-
```

Comment:
Isn't it good to have some separator for the different things here? I guess it's your linter which removes these, so maybe insert one or more `#` or something? What do you think?

Reply:
I agree, but I am done for today, this PR is not critical